sdk/src/main/java/io/dapr/client/DaprClientHttp.java

```diff
@@ -770,10 +773,18 @@ public Mono<List<ConfigurationItem>> getConfiguration(GetConfigurationRequest re
    * {@inheritDoc}
    */
   @Override
-  public Flux<List<ConfigurationItem>> subscribeToConfiguration(SubscribeConfigurationRequest request) {
+  public Flux<SubscribeConfigurationResponse> subscribeConfiguration(SubscribeConfigurationRequest request) {
     return DaprException.wrapFlux(new UnsupportedOperationException());
```

Comment:
There is a HTTP version added since 1.8. https://docs.dapr.io/reference/api/configuration_api/